protobuf/endpoints/batch_create_artifact.proto

```diff
@@ -2,9 +2,17 @@ syntax = "proto3";
 
 package endpoints;
 
+enum ArtifactType {
+  // TODO(hayden@preemo.io, 04/17/2023): should these be ARTFACT_TYPE... or ARTIFACTTYPE_...
+  ARTIFACT_TYPE_UNSPECIFIED = 0;
+  ARTIFACT_TYPE_PARAMS = 1;
+  ARTIFACT_TYPE_RESULT = 2;
+}
+
 // This config is included for potential future use.
 // In the future it may include part count, expiration, or other metadata.
 message CreateArtifactConfig {
+  ArtifactType type = 1;
```

Comment:
i'm not sure we're ever going to have dissimilar types, maybe this should just go on the request?

Reply:
For our protobuf interfaces we should prefer whichever option reduces the likelihood of having to change/update it in the future. So I think how you have it now makes perfect sense